tools/c7n_azure/c7n_azure/resources/appserviceplan.py

```diff
@@ -90,6 +90,14 @@ def _prepare_processing(self):
     def _process_resource(self, resource):
         model = models.AppServicePlan(location=resource['location'])
 
+        if resource['kind'] == 'functionapp':
```

Comment:
Should we just get rid of consumption plans when we get a list of all app service plan? (There is nothing we can configure\control, right?)

Reply:
We can actually still perform other generic actions like tag